tools/db/move-team/src/Work.hs

```diff
@@ -0,0 +1,183 @@
+module Work where
+
+import Brig.Types hiding (Client)
+import Cassandra hiding (Set)
+import Common
+import Conduit (mapC)
+import Control.Lens (view, _2, _3, _5)
+import Data.Aeson
+import Data.Conduit
+import qualified Data.Conduit.Combinators as C
+import Data.Conduit.Internal (zipSources)
+import qualified Data.Conduit.List as CL
+import Data.Id
+import qualified Data.Set as Set
+import Data.UUID
+import Galley.Data.Instances ()
+import Imports
+import Schema
+import System.Exit (ExitCode (ExitFailure, ExitSuccess), exitWith)
+import System.FilePath ((</>))
+import qualified System.IO as IO
+import qualified System.Logger as Log
+import System.Process (system)
+import Types
+
+deriving instance Cql Name
+
+assertTargetDirEmpty :: Env -> IO ()
+assertTargetDirEmpty Env {..} = do
+  files <- listDirectory envTargetPath
+  unless (Imports.null files) $ do
+    IO.putStrLn $ "Target directory " <> envTargetPath <> " is not empty."
+    exitWith (ExitFailure 1)
+
+runExport :: Env -> IO ()
+runExport env@Env {..} = do
+  ExitSuccess <- system $ "mkdir -p " <> show envTargetPath
+  assertTargetDirEmpty env
+  runTeam env
+  runGalleyTeamMembers env
+  runGalleyTeamConv env
+  readAllTeamUserIds env >>= runFullScans env
+
+runImport :: Env -> IO ()
+runImport env = do
+  IO.putStrLn "runImport"
+  importAllTables env
+
+runDebugExportFull :: Env -> IO ()
+runDebugExportFull env = do
+  assertTargetDirEmpty env
+  exportAllTablesFull env
+
+----------------------------------------------------------------------
+
+runTeam :: Env -> IO ()
+runTeam env@Env {..} = do
+  appendJsonLines (envTargetPath </> "galley.billing_team_member") (readGalleyBillingTeamMember env envTeamId)
+  appendJsonLines (envTargetPath </> "galley.team") (readGalleyTeam env envTeamId)
+  appendJsonLines (envTargetPath </> "galley.team_features") (readGalleyTeamFeatures env envTeamId)
+  appendJsonLines (envTargetPath </> "galley.team_member") (readGalleyTeamMember env envTeamId)
+  appendJsonLines (envTargetPath </> "galley.team_notifications") (readGalleyTeamNotifications env envTeamId)
+
+runGalleyTeamMembers :: Env -> IO ()
+runGalleyTeamMembers env@Env {..} =
+  IO.withBinaryFile (envTargetPath </> "galley.team_member") IO.WriteMode $ \outH ->
+    runConduit $
+      zipSources
+        (CL.sourceList [(1 :: Int32) ..])
+        (readGalleyTeamMember env envTeamId)
+        .| C.mapM (handleTeamMembers env)
+        .| sinkJsonLines outH
+
+handleTeamMembers :: Env -> (Int32, [RowGalleyTeamMember]) -> IO [RowGalleyTeamMember]
+handleTeamMembers env@Env {..} (i, members) = do
+  Log.info envLogger (Log.field "number of team members loaded: " (show (i * envPageSize)))
+  let uids = catMaybes $ fmap Id . view _2 <$> members
+
+  appendJsonLines (envTargetPath </> "brig.clients") (readBrigClients env uids)
+  appendJsonLines (envTargetPath </> "brig.connection") (readBrigConnection env uids)
+  appendJsonLines (envTargetPath </> "brig.connection") (readBrigIdMapping env uids)
```

Comment:
```suggestion
  appendJsonLines (envTargetPath </> "brig.id_mapping") (readBrigIdMapping env uids)
```

I wonder why this didn't cause any trouble during import?

Reply:
Ouch... Probably brig.id_mapping is empty. So it didn't append any bad data to `envTargetPath </> "brig.connection"`